pom.xml

```diff
@@ -178,7 +179,7 @@
     <distributionManagement>
         <repository>
             <id>maven-releases</id>
-            <url>https://repo.pluginwiki.us/repository/maven-releases/</url>
+            <url>http://ci.pluginwiki.us/plugin/repository/everything/</url>
```

Comment:
Can switch this back as everything the plugin needs is on the maven-releases repo